### PR TITLE
Update deps + code 

### PR DESCRIPTION
**Pull request by stevelacy**
- change to browserify transforms for stylus
- update deps
- remove morgan + unused deps

### CODE REVIEW

File: client/components/CircleIcon/index.coffee [hunk 1]
```diff
@@ -8,7 +10,7 @@ module.exports = component
       PropTypes.number
     ]).isRequired
     background: PropTypes.string
-
+  css: css
```
Comment (contra) [inline client/components/CircleIcon/index.coffee]: should come after displayName

**Comment by contra** [comment]
nice work - any other ideas on stuff to backport to the boilerplate?

File: client/components/CircleIcon/index.coffee (hunk 1 above)
Comment (stevelacy) [inline client/components/CircleIcon/index.coffee]: @contra it seems to be after displayName: https://github.com/wearefractal/product-boilerplate/pull/10/files#diff-258fef048fec1baa7e4da486f557f9d0R6
Unless I am mistaken by what you want after

**Comment by stevelacy** [comment]
Probably circleci/deployment, 404 page, maybe update the resource file with the custom endpoints if we have not already added that

**Comment by contra** [comment]
@stevelacy feel free to send PRs backporting good stuff whenever you have time

**Pull request merged**